cli/flox/doc/flox-activate.md

```diff
@@ -155,18 +155,29 @@ See [`manifest.toml(5)`](./manifest.toml.md) for more details on shell hooks.
 
 ## Variables used by `flox activate`
 
-`$FLOX_SHELL`
-:  When launching an interactive sub-shell, Flox launches the shell specified in
-   `$FLOX_SHELL` if it is set.
-   When printing a shell for sourcing in the current shell,
-   Flox will produce a script suitable for `$FLOX_SHELL` if it is set.
-
-`$SHELL`
-:  When launching an interactive sub-shell, Flox launches the shell specified in
-   `$SHELL` if it is set and `$FLOX_SHELL` is not set.
-   When printing a shell for sourcing in the current shell,
-   Flox will produce a script suitable for `$SHELL` if it is set
-   and `$FLOX_SHELL` is not set and Flox can't detect the parent shell.
+`$FLOX_SHELL`, `$FLOX_SHELL_NOCLOBBER`, `$SHELL`
+:  When activating an environment in "interactive" or "in-place" modes,
+   Flox will either launch an interactive sub-shell
+   or emit commands to configure an already-running (parent) shell.
+   In both of these cases Flox needs to know which shell to use,
+   and these variables are used to control the selection process.
+
+       * interactive: When launching an interactive sub-shell
```

Comment:
This is the same for command mode, right?

Reply:
OK will add mention of command mode